dev-utils/run-script.js

```diff
@@ -23,29 +23,18 @@
  *
  */
 
-const path = require('path')
-const testUtils = require('./test')
-const saucelabs = require('./saucelabs')
+const { join } = require('path')
+const { runKarma } = require('./test-utils')
 const { generateNotice } = require('./dep-info')
 
-function runUnitTests(launchSauceConnect, directory) {
-  var testConfig = testUtils.getTestEnvironmentVariables()
-  testConfig.karmaConfigFile = path.join(
-    __dirname,
-    '..',
-    directory,
-    './karma.conf.js'
-  )
-  if (launchSauceConnect !== 'false') {
```

Comment:
The `launchSauceConnect`  is useful when debugging issues with saucelabs, you can start the sauceconnect individually in another terminal and don't have to start it on each run!

Reply:
To be honest, you can do this via switch of a flag `sauceConnect` (true|false) in code since you are already debugging.

Reply:
IMO, passing it through the command line is easier since we don't have to change the code!

Reply:
I have reenabled the function as per your suggestion.